SabreTools.Models/SecuROM/MatroschkaPackage.cs

```diff
@@ -0,0 +1,41 @@
+using System.Collections.Generic;
+
+namespace SabreTools.Models.SecuROM
+{
+    /// <summary>
+    /// Securom Matroschka Package, a package contained in an executable as a section. Offered by SecuROM, its
+    /// main purpose seems to be managing some sort of SecuROM-related operation involving multiple temporary
+    /// files contained within the package. Observed in Release Control executables, Product Activation Revocation
+    /// executables, and in some regular Product-Activation-protected releases (such as the digital download releases
+    /// of Neverwinter Nights 2 and Test Drive Unlimited) where the game executable, paul.dll and other PA-related files
+    /// are stored in the matroschka package. 
+    /// </summary>
+    public class MatroschkaPackage
+    {
+        /// <summary>
+        /// Number of entries
+        /// </summary>
+        public int NumberOfEntries { get; set; }
+        
+        /// <summary>
+        /// Three unknown values only observed on RC matroschka sections. They only ever seem to be 0 or 1.
+        /// </summary>
```

Comment:
Nitpick: Also put a comment on the other 2 unknown values. If you hover over 2 or 3, you won't get the same hint otherwise.